components/filemanager/events.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Events
+page_title: FileManager Events
+description: Events in the FileManager for Blazor.
+slug: filemanager-events
+tags: telerik,blazor,upload,async,events
+published: true
+position: 20
+---
+
+# FileManager Events
+
+This article explains the events available in the Telerik FileManager for Blazor. They are grouped logically.
+
+* [CUD Events](#cud-events) - events related to Creating, Updating and Deleting items.
+    * [OnCreate](#oncreate)
+    * [OnUpdate](#onupdate)
+    * [OnEdit](#onedit)
+    * [OnDelete](#ondelete)
+* [Read Event](#read-event) - event related to obtaining data.
+* [Other Events](#other-events) - other events the grid provides.
+    * [OnModelInit](#onmodelinit)
+    * [OnDownload](#ondownload)
+
+## CUD Events
+
+The `OnCreate`, `OnUpdate` and `OnDelete` events let you get the data item that the user changed so you can transfer the user action to the actual data source.
+
+The `OnEdit` event let you respond to user actions - when they want to edit an item. See the [example](#example).
+
+### OnCreate
+
+The `OnCreate` event fires when a new item is created (new folder). Its event handler receives the updated `FileManagerCreateEventArgs` as an argument.
+
+### OnUpdate
+
+The `OnUpdate` event fires when a file is updated (rename finishes). Its event handler receives the updated `FileManagerUpdateEventArgs` as an argument.
+
+### OnEdit
+
+The `OnEdit` event fires when the user is about to enter edit mode for an existing item. Its event handler receives the updated `FileManagerEditEventArgs` as an argument.
+
+### OnDelete
+
+The `OnDelete` event fires when a file is deleted. Its event handler receives the updated `FileManagerDeleteEventArgs` as an argument.
+
+## Read Event
```

Comment:
There is no need to use two headings - Read and OnRead. Link to the new OnRead article for details, especially when there is no specific example here (I think there should be - with a lot of dummy data, which is generated with some loop).